deploy/catalog/templates/apiserver.yaml

```diff
@@ -42,7 +42,7 @@ spec:
         - name: apiserver-ssl
           mountPath: /var/run/kubernetes-service-catalog
           readOnly: true
-      {{ if eq (default .Values.storageType "etcd") "etcd" -}}
+      {{ if eq (.Values.storageType | default "etcd") "etcd" -}}
```

Comment:
The more straightforward way to write this is:

```
{{ if eq (default "etcd" .Values.storageType) "etcd" -}}
```

This avoids using a pipeline... which imo improves readability.

In the end, I don't care much... I'm going to have a complete re-vamp of the chart #517 dropping soon anyway.

LGTM